### PR TITLE
Radar chart

### LINKED ISSUE #69: Add radar chart to Monthly reports view
As a user, I should be able to 
-[ ] see a chart under the Monthly Reports tab in the Reports page that shows a radar chart of each user's data that corresponds to the table above it.

### PR DESCRIPTION
**Pull request by emackinnonTechtonic**
### Type of change:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features

### Why is this change required? What problem(s) does it solve?
This PR adds a radar chart to the monthly ranked reports tab view. Further styling is needed.
### Were there any challenges while implementing this feature? If so, how were they addressed?
N/A
### Where should the reviewer start? How can this be tested?
The user can navigate to the monthly ranked reports view
### Link issues
Closes #69

### CODE REVIEW

File: client/src/Views/Reports.vue [hunk 1]
```diff
@@ -82,6 +79,15 @@ export default {
...
     },
+    // handleRankedUserData: async function () {
+    //   const { selectedMonth, selectedYear } = this.createDateObj();
+    //   const formattedShouts = await getRankedByMonth(
+    //     "shouter",
+    //     selectedMonth,
+    //     selectedYear
+    //   );
+    //   this.shoutsToDisplay = rankedShoutersFormatter(formattedShouts);
+    // },
```
Comment (tgeeTechtonic) [inline client/src/Views/Reports.vue:90]: Do you need this?

File: client/src/Views/Reports.vue [hunk 2]
```diff
@@ -48,10 +49,11 @@ import {
...
 import Table from "../components/Table";
 import FillChart from "../components/FillChart";
+import RadarChart from "../components/RadarChart";
 
 export default {
   name: "Reports",
-  components: { Table, FillChart },
+  components: { Table, FillChart, RadarChart },
   data() {
     return {
```
Comment (broxburyTechtonic) [inline client/src/Views/Reports.vue:58]: Just FYI, this RadarChart will need to go in the new AllShouts component when my PR goes up

File: client/src/Views/Reports.vue (hunk 1 above)
Comment (broxburyTechtonic) [reply client/src/Views/Reports.vue:90]: In my PR, all this logic that is specific to a tab is broken out into it's proper component

**Review by broxburyTechtonic — APPROVED** [review]
Looks good! I will try to resolve the merge conflicts for my PR

File: client/src/Views/Reports.vue (hunk 1 above)
Comment (tgeeTechtonic) [reply client/src/Views/Reports.vue:90]: I mention it because it's commented out

Comment (broxburyTechtonic) [reply client/src/Views/Reports.vue:90]: It is still used I believe

**Review by tgeeTechtonic — APPROVED** [review]

**Pull request merged**